scss/_patterns_form-password-toggle.scss

```diff
@@ -0,0 +1,29 @@
+@import 'settings';
+
+@mixin vf-p-form-password-toggle {
+  .p-form-password-toggle {
+    display: flex;
+    justify-content: space-between;
+
+    .p-form-password-toggle__button {
```

Comment:
I propose using the standard p-button--base with a u-no-margin--bottom class on it to keep things on   the baseline grid. THen you only need to tell it to no go full-width on mobile.

Reply:
needs the `has-icon` class too